app/src/main/java/org/indywidualni/fblite/webview/MyAppWebViewClient.java

```diff
@@ -100,9 +100,13 @@ public void onPageFinished(WebView view, String url) {
         if (preferences.getBoolean("hide_news_feed", false))
             view.loadUrl("javascript:function addStyleString(str) { var node = document.createElement('style'); node.innerHTML = str; document.body.appendChild(node); } addStyleString('#m_newsfeed_stream{ display: none; }');");
 
+        // hide people you may know
+        if (preferences.getBoolean("hide_people", false))
+            view.loadUrl("javascript:function addStyleString(str) { var node = document.createElement('style'); node.innerHTML = str; document.body.appendChild(node); } addStyleString('._55wo, ._5rgr, ._5gh8, ._35au{ display: none; }');");
```

Comment:
I have found #pagelet_netego is for "People You May Know", but not sure if it is the same for Mobile.